hexa/pipelines/views.py

```diff
@@ -87,6 +92,58 @@ def credentials(request: HttpRequest) -> HttpResponse:
     )
 
 
+@require_POST
+@csrf_exempt
+def credentials2(request: HttpRequest) -> HttpResponse:
+    """This API endpoint is called by the Pipelines v2 to get credentials"""
+    auth_type, token = request.headers.get("Authorization", " ").split(" ")
+    if auth_type.lower() != "bearer":
+        return JsonResponse(
+            {"error": "Authorization header should start with 'bearer'"}, status=401
+        )
+    try:
+        data = Signer().unsign_object(token)
+    except BadSignature:
+        return JsonResponse({"error": "Token signature is invalid"}, status=401)
+
+    model = apps.get_model(data["app_label"], data["model"])
+    pipeline = get_object_or_404(model, pk=data["id"])
+    workspace = pipeline.workspace
+
+    env = {}
```

Comment:
Starting from here, everything should be copied as is from `workspaces.views.credentials`